[2024-06-03T17:39:28.006+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T17:39:28.025+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-12-01T00:00:00+00:00 [queued]>
[2024-06-03T17:39:28.032+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-12-01T00:00:00+00:00 [queued]>
[2024-06-03T17:39:28.033+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T17:39:28.045+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): games_etl> on 2021-12-01 00:00:00+00:00
[2024-06-03T17:39:28.050+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=236) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T17:39:28.051+0000] {standard_task_runner.py:63} INFO - Started process 240 to run task
[2024-06-03T17:39:28.052+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'games_etl', 'scheduled__2021-12-01T00:00:00+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpd2ughw_2']
[2024-06-03T17:39:28.053+0000] {standard_task_runner.py:91} INFO - Job 59: Subtask games_etl
[2024-06-03T17:39:28.086+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.games_etl scheduled__2021-12-01T00:00:00+00:00 [running]> on host a5a4421409ea
[2024-06-03T17:39:28.145+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='games_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-01T00:00:00+00:00'
[2024-06-03T17:39:28.146+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T17:39:28.147+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T17:39:29.005+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T17:39:29.012+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/chess_dag.py", line 19, in games_etl_task
    return get_monthly_games(execution_date)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/chess_etl_code.py", line 60, in get_monthly_games
    connection = BaseHook.get_connection('chess_project_s3')
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `chess_project_s3` isn't defined
[2024-06-03T17:39:29.018+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=chess_etl, task_id=games_etl, run_id=scheduled__2021-12-01T00:00:00+00:00, execution_date=20211201T000000, start_date=20240603T173928, end_date=20240603T173929
[2024-06-03T17:39:29.028+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 59 for task games_etl (The conn_id `chess_project_s3` isn't defined; 240)
[2024-06-03T17:39:29.067+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-03T17:39:29.084+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T17:39:29.087+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-03T22:24:37.734+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T22:24:37.750+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-12-01T00:00:00+00:00 [queued]>
[2024-06-03T22:24:37.756+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-12-01T00:00:00+00:00 [queued]>
[2024-06-03T22:24:37.757+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T22:24:37.766+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): games_etl> on 2021-12-01 00:00:00+00:00
[2024-06-03T22:24:37.773+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=30338) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T22:24:37.775+0000] {standard_task_runner.py:63} INFO - Started process 30340 to run task
[2024-06-03T22:24:37.774+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'games_etl', 'scheduled__2021-12-01T00:00:00+00:00', '--job-id', '134', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp0fugizwv']
[2024-06-03T22:24:37.775+0000] {standard_task_runner.py:91} INFO - Job 134: Subtask games_etl
[2024-06-03T22:24:37.810+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.games_etl scheduled__2021-12-01T00:00:00+00:00 [running]> on host f55fff24b127
[2024-06-03T22:24:37.876+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='games_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-01T00:00:00+00:00'
[2024-06-03T22:24:37.877+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T22:24:37.878+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T22:24:38.419+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-03T22:24:39.124+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T22:24:39.125+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T22:24:39.134+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=games_etl, run_id=scheduled__2021-12-01T00:00:00+00:00, execution_date=20211201T000000, start_date=20240603T222437, end_date=20240603T222439
[2024-06-03T22:24:39.152+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T22:24:39.166+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-03T22:24:39.167+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-08T19:50:14.415+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T19:50:14.429+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-12-01T00:00:00+00:00 [queued]>
[2024-06-08T19:50:14.434+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-12-01T00:00:00+00:00 [queued]>
[2024-06-08T19:50:14.434+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-08T19:50:14.446+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): games_etl> on 2021-12-01 00:00:00+00:00
[2024-06-08T19:50:14.454+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=30783) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-08T19:50:14.456+0000] {standard_task_runner.py:63} INFO - Started process 30793 to run task
[2024-06-08T19:50:14.455+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'games_etl', 'scheduled__2021-12-01T00:00:00+00:00', '--job-id', '310', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpoxyyrydd']
[2024-06-08T19:50:14.456+0000] {standard_task_runner.py:91} INFO - Job 310: Subtask games_etl
[2024-06-08T19:50:14.491+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.games_etl scheduled__2021-12-01T00:00:00+00:00 [running]> on host bf0ade909435
[2024-06-08T19:50:14.555+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='games_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-01T00:00:00+00:00'
[2024-06-08T19:50:14.556+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T19:50:14.556+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-08T19:50:15.117+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-08T19:50:15.777+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T19:50:15.778+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T19:50:15.786+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=games_etl, run_id=scheduled__2021-12-01T00:00:00+00:00, execution_date=20211201T000000, start_date=20240608T195014, end_date=20240608T195015
[2024-06-08T19:50:15.833+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T19:50:15.847+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-08T19:50:15.848+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-08T01:48:38.917+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-08T01:48:38.930+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-12-01T00:00:00+00:00 [queued]>
[2024-07-08T01:48:38.935+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-12-01T00:00:00+00:00 [queued]>
[2024-07-08T01:48:38.935+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-08T01:48:38.943+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): games_etl> on 2021-12-01 00:00:00+00:00
[2024-07-08T01:48:38.950+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=31670) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-08T01:48:38.951+0000] {standard_task_runner.py:63} INFO - Started process 31672 to run task
[2024-07-08T01:48:38.951+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'games_etl', 'scheduled__2021-12-01T00:00:00+00:00', '--job-id', '696', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpzb2rxgtp']
[2024-07-08T01:48:38.952+0000] {standard_task_runner.py:91} INFO - Job 696: Subtask games_etl
[2024-07-08T01:48:38.983+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.games_etl scheduled__2021-12-01T00:00:00+00:00 [running]> on host 6569f5155f0c
[2024-07-08T01:48:39.042+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='games_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-01T00:00:00+00:00'
[2024-07-08T01:48:39.043+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-08T01:48:39.044+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-07-08T01:48:39.551+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-07-08T01:48:40.218+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-08T01:48:40.220+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-08T01:48:40.226+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=games_etl, run_id=scheduled__2021-12-01T00:00:00+00:00, execution_date=20211201T000000, start_date=20240708T014838, end_date=20240708T014840
[2024-07-08T01:48:40.247+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-08T01:48:40.265+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-08T01:48:40.266+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
